hcl2template/types.build.hcp_packer_registry_test.go

```diff
@@ -57,6 +57,7 @@ func Test_ParseHCPPackerRegistryBlock(t *testing.T) {
 					Builder:        emptyMockBuilder,
 					Provisioners:   []packer.CoreBuildProvisioner{},
 					PostProcessors: [][]packer.CoreBuildPostProcessor{},
+					BuilderType:    "virtualbox-iso",
```

Comment:
Unless I'm wrong the function that reads it is only in the `hcp` package, if we don't add it here does the tests fail?

Reply:
Yes!